cms/djangoapps/contentstore/views/course.py

```diff
@@ -1367,12 +1389,21 @@ def group_configurations_list_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             group_configuration_url = reverse_course_url('group_configurations_list_handler', course_key)
             course_outline_url = reverse_course_url('course_handler', course_key)
-            configurations = GroupConfiguration.add_usage_info(course, store)
+            should_show_experiment_groups = are_content_experiments_enabled(course)
+            if should_show_experiment_groups:
+                experiment_group_configurations = GroupConfiguration.get_split_test_partitions_with_usage(course, store)
+            else:
+                experiment_group_configurations = None
+            content_group_configuration = GroupConfiguration.get_or_create_content_group_configuration(
+                course
+            ).to_json()
```

Comment:
This indentation is really confusing. Also, you have a variable named `content_group_configuration` that does not actually contain a `GroupConfiguration` object, but instead contains (presumably) a dict. Can you do something like:
```python
content_group_config = GroupConfiguration.get_or_create_content_group_configuration(course)
return render_to_response('group_configurations.html', {
    # other key/value pairs here...
    'content_group_configuration': content_group_config.json(),
})
```
And as an aside, `GroupConfiguration.get_or_create_content_group_configuration` is needlessly verbose, but I don't know if you can/should change that in this PR.